config/v1/types_infrastructure.go

```diff
@@ -144,6 +148,22 @@ type GCPPlatformStatus struct {
 	Region string `json:"region"`
 }
 
+// BareMetalPlatformStatus holds the current status of the BareMetal infrastructure provider.
+type BareMetalPlatformStatus struct {
+	// apiInternalIP is an IP address that can be used to contact the OpenShift API.
```

Comment:
```go
// apiInternalIP is an IP address to contact the kube-apiserver that can be used by 
components inside the cluster like kubelets using the infrastructure provider rather
than Kubernetes networking.  It matches the URL in Infrastructure.status.apiServerInternalURI.
??? It is the IP for a self-hosted load balancer in front of the API servers.
```
1. The load balancer is required to run on the kube cluster?
2. Must this IP be accessible from outside the cluster?